src/javascript/felipehfs-selectionSort.js

```diff
@@ -0,0 +1,19 @@
+
+function selectionSort(array) {
```

Comment:
@felipehfs add a single line comment for the description of the function

Reply:
Ok